### ISSUE TITLE
Question about evaluating the fine-tuned model on test set

### ISSUE THREAD

**Issue by Yutong-Lu** [IBM/molformer#21]
Hello, I was able to fine-tune the pre-trained model provided on the lipo dataset, and I obtained a model named last.ckpt at the end.

I wanted to evaluate the performance of this last checkpoint on my test set, so I was wondering how we could load this model (last.ckpt) and evaluate its performance on a test set.

Thank you for your help!

**Comment by ZeroDesigner** [comment]
> Hello, I was able to fine-tune the pre-trained model provided on the lipo dataset, and I obtained a model named last.ckpt at the end.
> 
> I wanted to evaluate the performance of this last checkpoint on my test set, so I was wondering how we could load this model (last.ckpt) and evaluate its performance on a test set.
> 
> Thank you for your help!

Have you solved the problem? I also meet the problem to evaluate it.
